src/util/hass-util.html

```diff
@@ -281,6 +281,29 @@
   }
 };
 
+window.hassUtil.sensorIcon = function (state) {
```

Comment:
`= (state) => {`

Reply:
Oh, we have ES6. Well that makes the code a lot nicer :)